docs/pages/enroll-resources/server-access/guides/host-user-creation.mdx

```diff
@@ -235,6 +237,138 @@ not already exist are created on the host. The `nginxrestarter` user is added to
+of an SSH session. Static host users require Teleport version >=16.3.0 on auth
+servers, SSH services, and `tctl`.
+
+### Step 1/3. Configure RBAC
+
+#### Label your server
+
+1. Access the server you enrolled with Teleport before beginning this guide and
+   open the Teleport configuration file, which is `/etc/teleport.yaml` by
+   default.
+
+1. Make the following change to the configuration file:
+
+   ```diff
+     labels:
+   +   app: "nginx"
+   ```
+
+1. Restart Teleport on the server.
+
+#### Define a Teleport role
+
+The following role specification allows users to log in as `nginxrestarter` on
+any matching server. Add this content to a file called `auto-users.yaml`:
+
+```yaml
+kind: role
+version: v5
+metadata:
+  name: auto-users
+spec:
+  allow:
+    logins: ["nginxrestarter"]
+    node_labels:
+      "app": "nginx"
+```
+
+Create the role:
+
+```code
+$ tctl create -f auto-users.yaml
+# role 'auto-users' has been created
+```
+
+#### Create a Teleport user
+
+1. Run the following command to create a Teleport user with the `auto-users`
+   role:
+
+   ```code
+   $ tctl users add demo-user --roles=auto-users --logins=nginxrestarter
+   ```
+
+1. Follow the instructions in your terminal to visit the Teleport Web UI and
+   create the user.
+
+### Step 2/3. Create the static host user
+
+Add this content to a file called `nginxrestarter.yaml`:
+
+```yaml
+kind: static_host_user
+metadata:
+  name: nginxrestarter
+spec:
+  matchers:
+    # Use either node_labels or node_labels_expression to select which servers
+    # to create the host user on. Only one is required.
+    - node_labels:
+      - name: app
+        values: ["nginx"]
+      node_labels_expression: 'labels["app"] == "nginx"'
+      # List of host groups the created user will be added to. Any that don't already exist are created.
+      groups: [ubuntu, nginx, other]
+      # List of entries to include in a temporary sudoers file created in /etc/sudoers.d
+      sudoers: [
+        # This line will allow the `nginxrestarter` user to run
+        # `systemctl restart nginx.service` as
+        # root without requiring a password.
+        # The sudoers entries will be prefixed with `nginxrestarter` in this case.
+        # sudoers file reference documentation: https://www.sudo.ws/docs/man/1.8.17/sudoers.man/
+        "ALL = (root) NOPASSWD: /usr/bin/systemctl restart nginx.service"
+      ]
+      # UID of the host user. Optional.
+      # uid: 1234
+      # GID of the host user. Optional.
+      # gid: 1234
+      # Default shell for the created user. Optional.
+      # default_shell: /bin/bash
+
+    # Add additional matchers if needed to configure nginxrestarter differently
+    # on different servers.
+    # - node_labels:
+      # ...
+```
+
+The name of the static host user resource must match the login of the user to
+be created. If your static host user has more than one matcher, a server can't
+be matched by more than one of them.
```

Comment:
Would this match to the first one in order?  Good if we can say that

Reply:
No. If more than one matcher applies to a server, it's an error and the static host user won't be applied at all.